code/atom.dm

```diff
@@ -680,7 +680,7 @@
 	return
 
 /atom/proc/attack_hand(mob/user as mob)
-	return
+	SEND_SIGNAL(src, COMSIG_ATOM_ATTACK_HAND, user)
```

Comment:
Good idea but out of the scope of this PR, especially considering that it would cause 386 errors.
Note that if we do that we should also establish some sort of a parent calling convention for these ubiquitous procs (where to call parent etc.).
Also in my opinion `attack_hand` should generally contain much less code than it usually does now and it should call other procs of the object so you can override those to change the behaviour more subtly. (A sidenote to the sidenote: Maybe we should even have some sort of a general system that could have multiple behaviours registered to attack_hand (and other such procs) that'd open a multicontext menu if there's more than one?)

Reply:
Also at the moment this PR makes no use of the ATTACK_HAND signal, but it doesn't hurt to have it for the future.